src/rulesets/Base/array.jl

```diff
@@ -39,6 +39,19 @@ function rrule(::typeof(hcat), A::AbstractArray, Bs::AbstractArray...)
     return hcat(A, Bs...), hcat_pullback
 end
 
+function rrule(::typeof(reduce), ::typeof(hcat), As::AbstractVector{<:AbstractVecOrMat})
+    function reduce_hcat_pullback(Ȳ)
```

Comment:
Could you use our `ΔY` notation? I know we're not entirely consistent in this package, but since we've [documented the standard](http://www.juliadiff.org/ChainRulesCore.jl/stable/FAQ.html#%CE%94x,-x,-dx-1), probably new rules should follow suit.

```suggestion
    function reduce_hcat_pullback(ΔY)
```

Reply:
Sure, I've taken the notation from `hcat` above: I guess `hcat` can be updated to follow the standard in a future PR.